tutorials/elixir-phoenix-on-google-app-engine.md

```diff
@@ -32,11 +35,16 @@ Before running this tutorial, take the following steps:
 
 2.  Enable billing for your project.
 
-3.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/). Make sure
+3.  [Enable the Google Cloud SQL Admin API in the Cloud Console](https://console.cloud.google.com/apis/library/sqladmin.googleapis.com/).
+
+4.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/). Make sure
     you [initialize](https://cloud.google.com/sdk/docs/initializing) the SDK
     and set the default project to the new project you created.
 
-    Version 175.0.0 or later of the SDK is required.
+    Version 227.0.0 or later of the SDK is required. If you have an earlier
+    version installed, you may upgrade it by running
```

Comment:
Suggestion: Make punctuation at the end of lead-in text consistent. This topic sometimes has no punctuation, sometimes a period, and sometimes a colon. I'd suggest using a colon.